pkg/jobs/jobspb/jobs.proto

```diff
@@ -1218,6 +1237,16 @@ message ChangefeedProgress {
     (gogoproto.customtype) = "github.com/cockroachdb/cockroach/pkg/util/uuid.UUID",
     (gogoproto.nullable) = false
   ];
+
+  // SpanLevelCheckpoint is the span-level checkpoint for a changefeed.
+  // It consists of a "map" from resolved timestamps to lists of spans
+  // with that resolved timestamp.
+  //
+  // On restart, the changefeed can restore the progress of spans in the
+  // checkpoint to its corresponding resolved timestamp, which will be higher
```

Comment:
this isn't actually implemented yet, right? Perhaps preface this note as a todo.